custom_components/zwave_mqtt/__init__.py

```diff
@@ -57,11 +58,16 @@ async def mark_platform_loaded(platform):
         if len(platforms_loaded) != len(PLATFORMS):
             return
 
-        hass.data[DOMAIN][entry.entry_id]["unsubscribe"] = await mqtt.async_subscribe(
-            hass, f"{TOPIC_OPENZWAVE}/#", async_receive_message
+        hass.data[DOMAIN][entry.entry_id][DATA_UNSUBSCRIBE].append(
```

Comment:
This change has already been merged in master. Something is weird with this branch now.

Reply:
Yeah, there are unrelated commits in this branch. Pleas *rebase* on latest master branch and remove any non relevant commits from your branch.

Reply:
Will do, I'll work on it

Reply:
I think that accomplished what we wanted, but let me know. Obviously I'm less familiar with certain parts of git

Reply:
Good!